datastore/redis.go

```diff
@@ -37,9 +37,23 @@ type BlockBuilderStatus string
 var (
 	RedisBlockBuilderStatusLowPrio     BlockBuilderStatus = ""
 	RedisBlockBuilderStatusHighPrio    BlockBuilderStatus = "high-prio"
+	RedisBlockBuilderStatusOptimistic  BlockBuilderStatus = "optimistic"
```

Comment:
Would it make sense to try to reuse `BlockBuilderStatusCode uint8` to reduce the amount of boilerplate?

Reply:
done! thanks. i added a string method to the common.BlockBuilderStatusCode type, so we can just use that an clean this up significantly.